service/src/main/java/bio/terra/profile/service/iam/model/SamAction.java

```diff
@@ -8,6 +8,7 @@ public enum SamAction {
   CREATE("create"),
   DELETE("delete"),
   UPDATE_BILLING_ACCOUNT("update_billing_account"),
+  UPDATE_METADATA("update_metadata"),
```

Comment:
This action is granted to `spend-profile` owners ([source](https://github.com/broadinstitute/sam/blob/b46ad2fc1a2cd1959a4e2623fcc5e8b63dc19c05/src/main/resources/reference.conf#L1100))